storage/src/test/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfigTest.java

```diff
@@ -52,6 +54,28 @@ public void testValidConfigs() {
         Assertions.assertEquals(expectedRemoteLogManagerConfig, remoteLogManagerConfig);
     }
 
+    @Test
+    public void testValidConfigsWithDefaultRemoteStorageManagerClass() {
+        String rsmPrefix = "__custom.rsm.";
+        String rlmmPrefix = "__custom.rlmm.";
+        Map<String, Object> rsmProps = Collections.singletonMap("rsm.prop", "val");
+        Map<String, Object> rlmmProps = Collections.singletonMap("rlmm.prop", "val");
+        RemoteLogManagerConfig expectedRemoteLogManagerConfig
+                = new RemoteLogManagerConfig(true, "dummy.remote.storage.class", "dummy.remote.storage.class.path",
+                "org.apache.kafka.server.log.remote.metadata.storage.TopicBasedRemoteLogMetadataManager", "dummy.remote.log.metadata.class.path",
```

Comment:
Could we replace the value with `DEFAULT_REMOTE_LOG_METADATA_MANAGER_CLASS_NAME`?

Reply:
Also, I think this test is basically identical to the `testValidConfigs` test, could we make it as `@ParameterizedTest`? Ex:
```
@ParameterizedTest
@ValueSource(booleans = {true, false})
testValidConfigs(boolean useDefaultRemoteStorageManagerClass) 
```

Reply:
Done.